CHANGES.md

```diff
@@ -11,6 +11,14 @@ updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
 
 -------------------------------------------------------------------------------
+## __cylc-8.1.1 (<span actions:bind='release-date'>Coming Soon</span>)__
+
+### Fixes
+
+[#5312](https://github.com/cylc/cylc-flow/pull/5312) - Force queues to be
+comma separated lists only. Tasks listed in queues will be treated as implicit
+in the same way as tasks in listed in runtime.
```

Comment:
I think this description is a bit misleading. Is this more accurate?:
```suggestion
[#5312](https://github.com/cylc/cylc-flow/pull/5312) - task names must be comma-separated in queue member lists. Any implicit tasks (i.e. with no task definition under runtime) assigned to a queue will generate a warning.
```
(Taking into account the suggestion below not to fail implicit tasks assigned to a queue).